### PR TITLE
add global hub release 2.11

### PR DESCRIPTION
**Pull request by yanmxa**
Signed-off-by: myan <myan@redhat.com>

### CODE REVIEW

File: ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-release-2.11.yaml [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+      REGISTRY_ORG: stolostron
+    workflow: ocm-ci-image-mirror
+- as: release-2100-agent-image-mirror
+  postsubmit: true
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: multicluster-global-hub-agent
+    env:
+      IMAGE_REPO: multicluster-global-hub-agent
+      IMAGE_TAG: v1.0.0
```
Comment (clyang82) [inline ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-release-2.11.yaml:192]: `s/v1.0.0/v1.2.0`

File: ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-release-2.11.yaml [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+  postsubmit: true
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: multicluster-global-hub-manager
+    env:
+      IMAGE_REPO: multicluster-global-hub-manager
+      IMAGE_TAG: v1.0.0
+      REGISTRY_ORG: stolostron
+    workflow: ocm-ci-image-mirror
+- as: release-2100-agent-image-mirror
```
Comment (clyang82) [inline ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-release-2.11.yaml:185]: `s/release-2100/release-2110/`

File: ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-release-2.11.yaml [hunk 3]
```diff
@@ -0,0 +1,198 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
```
Comment (clyang82) [inline ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-release-2.11.yaml:5]: can we use `12`? just have a try. ocp 4.8 is very old.

**Review by clyang82 — COMMENTED** [review]
You need to update https://github.com/openshift/release/blob/659d109b382f82a61f8c94d63259cf29cb2c189a/ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-main.yaml#L182 to `release-2.11` branch.

File: ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-release-2.11.yaml (hunk 1 above)
Comment (yanmxa) [reply ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-release-2.11.yaml]: Done.

File: ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-release-2.11.yaml (hunk 2 above)
Comment (yanmxa) [reply ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-release-2.11.yaml]: Done

File: ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-release-2.11.yaml (hunk 3 above)
Comment (yanmxa) [reply ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-release-2.11.yaml]: Done.

**Pull request merged**